programs/vote/src/vote_state/mod.rs

```diff
@@ -958,10 +959,24 @@ pub fn withdraw<S: std::hash::BuildHasher>(
         .lamports()?
         .checked_sub(lamports)
         .ok_or(InstructionError::InsufficientFunds)?;
+    let received_credits_previous_epoch = match vote_state.epoch_credits.len() {
+        0 => false,
+        _ => {
+            let maybe_last_epoch_credits = vote_state.epoch_credits.last();
+            match maybe_last_epoch_credits {
+                None => false,
+                Some(last_epoch_credits) => (last_epoch_credits.1 - last_epoch_credits.2) > 0,
```

Comment:
```suggestion
                Some((epoch_credits, previous_epoch_credits)) => epoch_credits.saturating_sub(previous_epoch_credits) > 0,
```
I think naming the tuple elements is a little more clear.
Also, we only use checked math in stake and vote programs. Current credits should never be smaller than previous credits, but... safety first ;)